get-all.sh

```diff
@@ -6,8 +6,6 @@ echo
 
 owner="Vanilla-OS"
 
-# set -x
```

Comment:
I left that on purpose to easily enable/disable debugging

Reply:
Added it back.